src/pages/en/editor-setup.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Editor Setup
+description: Set up your editor to build with Astro.
+---
+You can build your Astro site with any code editor, installed locally or in the cloud!
+
+[VS Code](https://code.visualstudio.com) is a popular code editor that can be dowloaded locally, and is also used by online platforms such as [GitHub.dev](https://github.dev), [GitHub Codespaces](https://github.com/features/codespaces) and [Gitpod](https://gitpod.io).
+
+
+## VS Code Extension
+
+Astro has its own [Astro VS Code Extension on Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode) that you can install in any of these VS Code environments for .`astro` syntax highligting, code completion, and more!
+
+### Full List of Features
+- [Go to Definition](https://code.visualstudio.com/docs/editor/editingevolved#_go-to-definition)
+- Code hover hints 
+- Code completion
+- Function signatures
+- Syntax highlighting
+- Code folding
+- Emmet
+
+After [setting up VS Code](https://code.visualstudio.com/Docs/setup/setup-overview), to get started building with Astro:
+
+1. Navigate to the "Extensions" tab ("Ctrl-Shift-X")
+2. Search for "astro" in the VS Code Marketplace
+3. Install the [Astro VS Code Extension](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode)
+
+## Other Astro language extensions
+
+You can also get Astro language support for other code editors using:
+
+[Astro VS Code Extension on Open VSX](https://open-vsx.org/extension/astro-build/astro-vscode) - the complete VS Code Extension, available on the Open VSX registry for open platforms like Gitpod.
```

Comment:
A lot of users initially asked for this because they use "free" versions of VS Code such as VSCodium, it'd be cool to mention those in addition to GitPod

Reply:
Absolutely!